lib/orderbook/types.ts

```diff
@@ -15,6 +15,7 @@ export type PlaceOrderResult = {
   internalMatches: OwnOrder[];
   swapSuccesses: SwapSuccess[];
   remainingOrder?: OwnOrder;
+  swapFailures: PeerOrder[];
```

Comment:
Minor: should jump this one line up, next to `swapSuccesses`. also in other sections in this PR.

Reply:
OK I was actually thinking about doing the same, so I'll go ahead with that since you had the same thought.